### PR TITLE
updated /desktop/features to vbt

### LINKED ISSUE #1659: /desktop/features/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* updated /desktop/features to vbt
* I can’t get the gaming section animation right… someone needs to help, it is flashing at the top of the page
* removed the open_source_phone include and embedded it
* put email on a white strip and got rid of the animated gif

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop/features)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-features.png) and [demo](http://www.ubuntu.com-1659-desktop-features.demo.haus/desktop/features)

## Issue / Card

Fixes #1659

### CODE REVIEW

**Comment by yaili** [comment]
- [ ] Every time I go to the page the gaming image fades in in the hero
- [ ] Remove border above "Photos"
- [ ] Add border above "Videos"
- [ ] Can the image in "Smarter searching" be full width, looks aligned left
- [ ] The hero image is too much to the right in large screens
- [ ] The text in "Gaming" should not become dark in s screens
- [ ] Might be better to hide the queen's image bit in s screen
- [ ] There is too much space above the image in "Organise your photos" in s screen

**Comment by pmahnke** [comment]
@yaili 'too much space' was actually white on white text!

**Comment by pmahnke** [comment]
@anthonydillon can you help with the animation on the Gaming section?

**Comment by yaili** [comment]
- [ ] Have you looked at the image in Web browsing when the viewport starts to shrink? The browser goes over the left icons after a while, and when the queen disappears the text goes over the icons. I think we need to fix this. Would an empty column help, or are they not rendered in s screen?
- [ ] Animation in hero still not fixed - just here for reference

File: templates/desktop/features.html [hunk 1]
```diff
@@ -30,266 +30,262 @@
...
-        </div>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6 suffix-1">
+      <h2>Email</h2>
+      <p>Ubuntu comes with Thunderbird, Mozilla&rsquo;s popular email application, so you&rsquo;ll have fast desktop access to your email. No matter which email services you use; Microsoft Exchange, Gmail, Hotmail, POP or IMAP, email just works.</p>
     </div>
-</div>
+    <div class="col-6 u-vertically-center">
+      <img src="{{ ASSET_SERVER_URL }}20fdbf70-features-email.jpg" class="" alt="" height="398" width="520" />
```
Comment (anthonydillon) [inline templates/desktop/features.html]: Remove empty class.

File: templates/desktop/features.html [hunk 2]
```diff
@@ -30,266 +30,262 @@
...
-        <div class="five-col">
-            <div>
-                <h2>Photos</h2>
-                <p>Ubuntu is full of free apps to help you enjoy, manage, edit and share your photos &mdash; whatever you use to take them. With fantastic support for cameras and phones, you won&rsquo;t need any extra drivers to get up and running.</p>
-            </div>
+<section class="p-strip--light is-deep">
+  <div class="row">
+    <div class="u-equal-height u-vertically-center">
+      <div class="col-5 suffix-1">
+        <div>
```
Comment (anthonydillon) [inline templates/desktop/features.html]: Does this need this extra `div`?

File: templates/desktop/features.html [hunk 3]
```diff
@@ -30,266 +30,262 @@
...
+          <h2>Videos</h2>
+          <p>Watch all your favourite content on Ubuntu with apps for playing, managing and sharing your videos. Edit your movies with PiTiVi and then watch them in Movie Player &mdash; or add VLC and OpenShot from the Ubuntu Software Centre, for compatibility with even more file formats.</p>
         </div>
-        <span class="photo-credit not-for-small smaller"></span>
+      </div>
+      <div class="col-7 u-hidden--small">
+        <img src="{{ ASSET_SERVER_URL }}5ea4959b-desktop-features-videos.jpg?w=542" alt="Video" />
```
Comment (anthonydillon) [inline templates/desktop/features.html]: Please improve alt or leave it empty.

File: templates/desktop/features.html [hunk 4]
```diff
@@ -30,266 +30,262 @@
...
-                <li>
-                    <img src="{{ ASSET_SERVER_URL }}786ed546-blender.svg" alt="Blender icon" />
-                </li>
-            </ul>
-        </div>
+<section class="p-strip is-dark p-gaming">
+  <div class="row">
+    <div class="col-12 prefix-6 p-gaming__content">
+      <h2>Gaming</h2>
+      <p>From Sudoku to shoot-em-ups, we&rsquo;ve got loads of games that&rsquo;ll keep you busy for hours.  There are thousands of games available for Ubuntu, including titles from the Unity and Steam platforms.  Pick from critically acclaimed titles such as Dota 2, Civilization V, Kerbal Space Program, Football Manager 2015 and Borderlands&nbsp;2.</p>
```
Comment (anthonydillon) [inline templates/desktop/features.html]: Let's remove `Football Manager 2015` it's old and hard to maintain.

File: templates/desktop/features.html [hunk 5]
```diff
@@ -30,266 +30,262 @@
...
-    <div class="strip-inner-wrapper">
-        <div class="prepend-six six-col last-col row-gaming__content">
-            <h2>Gaming</h2>
-            <p>From Sudoku to shoot-em-ups, we&rsquo;ve got loads of games that&rsquo;ll keep you busy for hours.  There are thousands of games available for Ubuntu, including titles from the Unity and Steam platforms.  Pick from critically acclaimed titles such as Dota 2, Civilization V, Kerbal Space Program, Football Manager 2015 and Borderlands&nbsp;2.</p>
+<section class="p-strip--light">
+  <div class="row p-divider">
+    <div class="col-6 p-divider__block">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}095a11c1-picto-projectswelove-orange.svg">
```
Comment (anthonydillon) [inline templates/desktop/features.html]: Missing alt

File: templates/desktop/features.html [hunk 6]
```diff
@@ -30,266 +30,262 @@
...
-    <div class="strip-inner-wrapper">
-        <div class="eight-col">
-            <div>
-                <h2>Smarter searching</h2>
-                <p>By enabling online search, smart filters make it faster and easier to find the content you need, whether it’s stored on your computer or on the web. Type any query into the Dash home and the Smart Scopes server will determine which categories of content are the most relevant to your search, returning only the best results. And it learns from your past searches making it increasingly accurate over time.</p>
-            </div>
+    <div class="col-6 p-divider__block">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}faf8037a-picto-canonical.png">
```
Comment (anthonydillon) [inline templates/desktop/features.html]: Missing alt

File: templates/desktop/features.html [hunk 7]
```diff
@@ -30,266 +30,262 @@
...
+          <h3 class="p-heading-icon__title">Backed by Canonical</h3>
         </div>
-        <img src="{{ ASSET_SERVER_URL }}bcb09a5b-desktop-features-smarter-searching.jpg?op=region&amp;rect=140,0,910,300" class="touch-border not-for-small" alt="Unity search" width="910" />
+      </div>
+      <p>Canonical is the global software vendor that provides commercial, design and engineering support to the Ubuntu project. Today, our hardware enablement team supports the pre-installation of Ubuntu on more than 10% of all new PCs shipped, worldwide.</p>
+      <p><a href="http://partners.ubuntu.com/partnering-with-us">Learn more about how we support Ubuntu&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/desktop/features.html]: external link please

**Comment by anthonydillon** [comment]
@pmahnke Fixed the animation and added card-overlay on some hard to read content.

File: templates/desktop/features.html (hunk 1 above)
Comment (pmahnke) [inline templates/desktop/features.html]: you fixed this

File: templates/desktop/features.html (hunk 2 above)
Comment (pmahnke) [inline templates/desktop/features.html]: removed

File: templates/desktop/features.html (hunk 3 above)
Comment (pmahnke) [inline templates/desktop/features.html]: done

File: templates/desktop/features.html (hunk 6 above)
Comment (pmahnke) [inline templates/desktop/features.html]: added

File: templates/desktop/features.html (hunk 7 above)
Comment (pmahnke) [inline templates/desktop/features.html]: added

**Comment by pmahnke** [comment]
@yaili fixed 'Web browsing' by putting the text on an overlay and prefixing the queen... how is that?

**Pull request merged**